zjit/src/hir.rs

```diff
@@ -1468,6 +1489,7 @@ impl Function {
             &ArrayExtend { left, right, state } => ArrayExtend { left: find!(left), right: find!(right), state },
             &ArrayPush { array, val, state } => ArrayPush { array: find!(array), val: find!(val), state },
             &CheckInterrupts { state } => CheckInterrupts { state },
+            &EntryPoint { jit_entry } => EntryPoint { jit_entry },
```

Comment:
as with LoadPC

Reply:
fixed in https://github.com/ruby/ruby/pull/14643/commits/1b6528aa09ded8e7575287994e52351f8b50783e